src/pages/test-report/cat-b/test-report.cat-b.page.ts

```diff
@@ -139,21 +133,22 @@ export class TestReportCatBPage extends PracticeableBasePageComponent {
         select(getTests),
         select(getCurrentTest),
         select(getTestData),
-        select(hasManoeuvreBeenCompleted),
-      ),
-      isLegalRequirementsValid$: this.store$.pipe(
-        select(getTestReportState),
-        select(isLegalRequirementsValid),
+        select(hasManoeuvreBeenCompletedCatB),
       ),
       isEtaValid$: this.store$.pipe(
         select(getTestReportState),
         select(isEtaValid),
       ),
-      catBLegalRequirements$: this.store$.pipe(
+      testData$: this.store$.pipe(
         select(getTests),
         select(getCurrentTest),
         select(getTestData),
-        select(getCatBLegalRequirements),
+      ),
+      testRequirements$: this.store$.pipe(
+        select(getTests),
+        select(getCurrentTest),
```

Comment:
Quick refactor whilst we're in here to create a `const currentTest$` instead of piping off `this.store$` excessively

Reply:
done